client/my-sites/email/email-providers-stacked-comparison/email-provider-stacked-card/index.tsx

```diff
@@ -3,8 +3,10 @@ import { useBreakpoint } from '@automattic/viewport-react';
 import classnames from 'classnames';
 import { FunctionComponent, useState } from 'react';
```

Comment:
Nit: this should be a type import for `FunctionComponent`.
```suggestion
import { useState } from 'react';
import type { FunctionComponent } from 'react';
```